app/tests/test_training.py

```diff
@@ -648,6 +651,97 @@ def test_difficulty_form(self):
 
                 # should not create any new attempts
                 self.assertEqual(Attempt.query.count(), 1)
+    
+    def test_correct_fill_in_the_blank_question_submission(self):
+        self.course.users.append(self.u2)
+        db.session.commit()
+
+        for user in [self.u1, self.u2]:
+            client = self.app.test_client(user=user)
+            response = client.post(url_for('user_views.test',
+                                                course_name="test-course",
+                                                mission_id=1),
+                                        data={
+                                            "question_id": str(self.fitb_question.id),
+                                            "response": "Answer1, Answer2",
+                                            "booger": "MEOW",
```

Comment:
I'm guessing you got this from looking at another example. Not sure why I had that in there, probably some sanity test thing that should have been removed. Let's not duplicate that embarrassing code though so remove that here 😂